app/invocation/invocation.tsx

```diff
@@ -264,6 +264,18 @@ export default class InvocationComponent extends React.Component<Props, State> {
               value={this.getBuildLogs()}
               loading={this.areBuildLogsLoading()}
               expanded={activeTab == "log"}
+              fullLogsFetcher={() => {
```

Comment:
Should we pull this out into a function? Seems like embedding this much JS into a render function seems like an antipattern.